### DISCUSSION TITLE
Sopra x MTE by Octo

### DISCUSSION THREAD

**Discussion by ddang-git** [teamg4it/g4it#17] [General]
Création du fil de conversation avec Sopra.

**Comment by bleroux-mte** [comment]
Bonjour, nous rencontrons au déploiement un problème de connexion à la BDD ecoSQL (sans doute moins permissive que nos bases de données postgreSQL locales) lié à un manque de slots.

J'ai créé une issue ici : https://github.com/G4ITTeam/g4it/issues/19

**Reply by ddang-git** [reply]
CC @denis-lemercier cf. message précédent.
